tsconfig.json

```diff
@@ -1,6 +1,6 @@
 {
   "compilerOptions": {
-    "target": "es5",
+    "target": "es2020",
```

Comment:
in `eslintconfig` we have `target: es2021` but here `es2020`, is it ok

Reply:
Yes, thanks , i'll fix